gradle-scalatest.plugin

```diff
@@ -0,0 +1,21 @@
+buildscript {
+	repositories {
+		maven {
+			url "https://plugins.gradle.org/m2/"
+		}
+	}
+	dependencies {
+		classpath "com.github.maiflai:gradle-scalatest:0.8"
+	}
+}
+
+if (!project.plugins.findPlugin(com.github.maiflai.ScalaTestPlugin)) {
```

Comment:
Thanks for the insights. I'm convinced, removed in https://github.com/openzipkin/zipkin/commit/b4e7113e437f0c47731f28f25f3f89dcce6194f7